docs/docs/community/monorepo-structure.md

```diff
@@ -42,7 +42,7 @@ This is the WebSocket NestJs server which connects to the widget and provides re
 
 This is the Novu admin panel which is used to visually communicate with the API. You can configure templates, manage content, enable or disable notifications, visually track the notification activity feed, etc...
 
-The `WEB` project is a create-react-app built, well, with React. 😄
+The `WEB` project is a create-react app built, well, with React. 😄
```

Comment:
```suggestion
The `WEB` project is a create-react-app built, well, with React. 😄
```
I think we can leave it like create-react-app because it is related to react npm package https://github.com/facebook/create-react-app